### PR TITLE
Project - Alejandra

### PR DESCRIPTION
**Pull request by alejandracosta**

### CODE REVIEW

**Comment by alejandracosta** [comment]
My project is ready for review! 
Thanks

File: _projects/2023/100385252/100385252.Rmd [hunk 1]
```diff
@@ -0,0 +1,660 @@
+---
+title: "Life expectancy increased in all countries of the world"
+description: "The aim of this project is to reproduce, layer by layer, a chart on Life Expectancy evolution from Our World in Data"
+author: Alejandra Costa
+cataegories: "2023"
```
Comment (Enchufa2) [inline _projects/2023/100385252/100385252.Rmd]: typo

File: _projects/2023/100385252/100385252.Rmd [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+    toc: true
+---
+
+Across the world, people are leaving longer. In 1800, the average global life expectancy was 32 years. Nevertheless, by 2012 this figure had more than doubled, reaching 70 years. But when and where has this improvement in life expectancy occurred?
+
+In order to understand such worldwide increase in longevity, we can look at the "Life Expectancy of the World Population in 1800, 1950 and 2012" chart, sourced from Our World in Data. Using information on population and life expectancy at birth from Gapminder.org, the mentioned chart shows how living standards have increased over the long run through the visualization of life expectancy evolution over the world.
+
+Accordingly, this post aims to replicate the provided chart, as well as to propose potential enhancements and alternatives.
+
+## 1. Getting and preparing the data
```
Comment (Enchufa2) [inline _projects/2023/100385252/100385252.Rmd]: Please remove the numbering from titles to keep the style of the rest of the projects.

File: _projects/2023/100385252/100385252.Rmd [hunk 3]
```diff
@@ -0,0 +1,660 @@
...
+
+```{r echo = FALSE}
```
Comment (Enchufa2) [inline _projects/2023/100385252/100385252.Rmd]: Do not use this. The reader must know which libraries you used!

File: _projects/2023/100385252/100385252.Rmd [hunk 4]
```diff
@@ -0,0 +1,660 @@
...
+```
+
+### Appearance adjustments
+
+We proceed by adding the corresponding title, subtitle and caption to the plot, as well as adjusting the graph appearance, where the font used is "Playfair Display", downloadable from Google Fonts.
+
+```{r fig.height = 5, fig.width = 7, fig.align = "center"}
+
+sysfonts::font_add_google("Playfair Display") 
+showtext::showtext_auto()
```
Comment (Enchufa2) [inline _projects/2023/100385252/100385252.Rmd]: Instead of calling this line, it's better to add `fig.showtext=TRUE` to all chunks with graphic output.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Nice work! I left you some comments with some fixes and improvements. As always, just modify, re-knit, add, commit and push. Marking this as a draft PR. Click "ready for review" again when you want me to make another pass.

File: _projects/2023/100385252/100385252.Rmd [hunk 5]
```diff
@@ -0,0 +1,670 @@
...
+    self_contained: false
+    toc: true
+---
+
+Across the world, people are leaving longer. In 1800, the average global life expectancy was 32 years. Nevertheless, by 2012 this figure had more than doubled, reaching 70 years. But when and where has this improvement in life expectancy occurred?
+
+In order to understand such worldwide increase in longevity, we can look at the "Life Expectancy of the World Population in 1800, 1950 and 2012" chart, sourced from Our World in Data. Using information on population and life expectancy at birth from Gapminder.org, the mentioned chart shows how living standards have increased over the long run through the visualization of life expectancy evolution over the world.
+
+Accordingly, this post aims to replicate the provided chart, as well as to propose potential enhancements and alternatives.
+
```
Comment (Enchufa2) [inline _projects/2023/100385252/100385252.Rmd:22]: Please show the original plot here. See the example project.

File: _projects/2023/100385252/100385252.Rmd [hunk 6]
```diff
@@ -0,0 +1,670 @@
...
+Additionally, information regarding the world region to which each country belongs is included, providing a more comprehensive plot and allowing the identification of potential patterns between areas. Also, with this kind of chart, by observing the slopes, we can observe the level of inequality across countries for each year in terms of living standards and life expectancy. For instance, in 1950, we observe a steep slope, meaning that there was a high inequality between countries, despite life expectancy levels were higher than in 1800.
+
+In order to create the proposal, the chart is plotted using geom_jitter() along with faced_grid(), therefore, an independent subplot is displayed for each of the years, where countries are represented by the data points.
+
+```{r fig.height = 5, fig.width = 8, fig.align = "center", fig.showtext = TRUE}
+
+colors_faced <- c("orange", "#7570b3","#7fc97f", "#43a2ca")
+
+data_regions <- read.csv(file = "Geographies.csv", sep = ";") |>
```
Comment (Enchufa2) [inline _projects/2023/100385252/100385252.Rmd:544]: This CSV is missing in the PR.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
A couple more comments. Let me know when this is ready again.

**Comment by Enchufa2** [comment]
Great work, congrats! Merging with a few fixes.

**Review by Enchufa2 — APPROVED** [review]

**Comment by Enchufa2** [comment]
And here's your work: https://csslab.uc3m.es/dataviz/projects/2023/100385252/

**Pull request merged**